internal/protocols/fm1200/types.go

```diff
@@ -140,6 +141,7 @@ func (r *Record) ToProtobufDeviceStatus() *types.DeviceStatus {
 	info.Position.Course = float32(r.Record.GPSElement.Angle)
 	info.Position.Satellites = int32(r.Record.IOElement.Properties1B[TIO_GSMSignal])
 	info.Temperature = float32(r.Record.IOElement.Properties4B[TIO_DallasTemperature])
+	info.AmbientTemperature = float32(r.Record.IOElement.Properties1B[TIO_AmbientAirTemperature])
```

Comment:
either ambient temperature or Dallas temperature should be parsed, not both diffrently

Reply:
like this @beastvaibhv 
```suggestion
 info.Temperature = float32(r.Record.IOElement.Properties4B[TIO_DallasTemperature]) ||  float32(r.Record.IOElement.Properties1B[TIO_AmbientAirTemperature])
```